world/World.hpp

```diff
@@ -0,0 +1,29 @@
+#include <unordered_map>
+
+#include "LevelData.hpp"
+#include "Level.hpp"
+
+#ifndef WORLD_HPP
+#define WORLD_HPP
+
+class World
```

Comment:
In the World file from cubic-server/World.hpp it's "CUBICSERVER_WORLD_HPP" not "WORLD_HPP" but if you think it will be preferable to change it I will do it

Reply:
Well, what's the difference between the two ? How are they gonna be used ? I mean, there is two world class here, so I am a bit confused.

The one in cubic server host the dimensions & entities
(btw, why are entites in the world and not in the dimension ?)

Reply:
I don't know too...
I think we need to talk about that quickly